### ISSUE TITLE
Color mode notification bar

### ISSUE THREAD

**Issue by agn4965** [anthonycr/Lightning-Browser#183]
With color mode disabled I think the notification bar would look better if it was black instead of translucent, with the status bar being light gray and the text white it isn't very legible.

**Comment by shorty66** [comment]
I think this issue refers to the case when "hide statusbar while browsing" is enabled.

**Comment by agn4965** [comment]
I'm not sure what you mean by that exactly but I don't have that option enabled.

**Comment by anthonycr** [comment]
Here's what it looks like for me and it seems readable. Can you confirm you're seeing the same thing on 5.0.2 and that it looks the same? The gray statusbar background isn't that dark but the status bar text is still very legible to me... It's actually darker than the status bar for the Google Drive app.

![screenshot_2015-02-04-13-59-23](https://cloud.githubusercontent.com/assets/1966779/6047225/62ff2132-ac76-11e4-877e-dc7e363fd3e1.png)

**Comment by agn4965** [comment]
Yes that's what it looks like. All the other apps I use have it turned black.

**Comment by karolba** [comment]
I think he is right. As for someone who sees that notification bar's color for the first time, first impression is that it looks quite ugly

**Comment by shorty66** [comment]
Okay, sorry. I menat something else:
When i turn on the option "hide statusbar while browsing" the transparent-background statusbar overlays the grey adressbar - not very well readable, too:
![screenshot_2015-02-04-20-13-45](https://cloud.githubusercontent.com/assets/10434328/6047531/90bc0c28-acaa-11e4-984e-c4298d64be2a.png)

**Comment by anthonycr** [comment]
@agn4965 okay, well in Lollipop there is a new design style where the app sets the status bar to be a darker color of the toolbar (the thing the search bar is placed on), I'm not going to make it black as that is against the guidelines, but I can make it a darker gray that is more readable.

@shorty66 yeah that definitely should not be happening. That definitely needs to be fixed. Looks like you are on 4.4?

**Comment by shorty66** [comment]
Yes, i am perfectly fine with that solution.

**Comment by agn4965** [comment]
Also this happens with the sidebars, but it's not very annoying.
![http://img.pleb.me/Screenshot_2015-02-04-20-12-11.png](http://img.pleb.me/Screenshot_2015-02-04-20-12-11.png)

The Mail app also uses a black bar...
![http://img.pleb.me/Screenshot_2015-02-04-20-18-30.png](http://img.pleb.me/Screenshot_2015-02-04-20-18-30.png)
